docs/eps/EVM_AGILE.md

```diff
@@ -0,0 +1,49 @@
+# EVM Agile - Dr. Down
+
+## Histórico de Revisões
+| Data | Versão | Descrição | Autores |
+| --- | --- | --- | --- |
+| 28/03/2018 | 0.0.1 | Criação do documento | João Pedro Sconetto |
+
+## EVM Agile
+
+O __EVM__ (_Earned Value Management_ ou Gerenciamento de Valor agregado) se trata de uma técnica da gestão tradicional de projetos que é usada para medir a saúde de um projeto, essa técnica foi adaptada para projetos agéis afim de auxiliar a equipe de gestão a verificar métricas e avaliar entregas em projetos que seguem esta metodologia, e que enfretam alguns problemas como:
```

Comment:
Sugestões de alteração:
"O __EVM__ (_Earned Value Management_ ou Gerenciamento de Valor agregado) se trata de uma técnica da gestão tradicional de projetos que é usada para medir a saúde de um projeto ~~, e~~ _. Essa_ técnica foi adaptada para projetos agéis ~~afim~~ _a fim_ de auxiliar a equipe de gestão a verificar métricas e avaliar entregas em projetos que seguem esta metodologia, e que enfretam alguns problemas como"